tools/deployment_options.py

```diff
@@ -100,7 +100,7 @@ def get_tag(image_fqdn):
     return image_fqdn.split(":")[-1]
 
 
-def get_image_override(deployment_options, short_image_name, env_var_name):
+def get_image_override(deployment_options, short_image_name, env_var_name, repo="ocpmetal"):
```

Comment:
Nit but the term for this is "org" instead of "repo"

For images the pattern is `<registry>/<org>/<repo>:<tag>`